Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/lucene/impl/LuceneUtility.java

```diff
@@ -392,33 +393,23 @@ public void indexConcepts() throws LuceneException, IllegalArgumentException, Il
             if (search != null) {
```

Comment:
I though this was already changes:
line 388/389 you have
```
ConceptEntry con = new ConceptEntry();
java.lang.reflect.Field[] fields = con.getClass().getDeclaredFields();
```
you don't have to create an object first, you can get the fields from the class itself.